src/xmpp/index.coffee

```diff
@@ -1,9 +1,19 @@
 'use strict'
 name = 'jm.xmpp'
-angular.module(name, []).factory 'XmppService', [
+
+xmppModule = angular.module name, []
+xmppModule.constant 'XMPPEvents', require 'jitsi-meet/service/xmpp/XMPPEvents'
```

Comment:
@watsoncj, what do you think about prefixing the angular component names with the module (so jm.xmpp.XMPPEvents)?  And if we do that, maybe put any wrapper services like MembersService in a separate module to show what is added in this project (ajm.xmpp.MembersService).  Also, I'm going to rename MembersService to MemberService